### PR TITLE
modify upload location to xcat.org for xCAT 2.7

### PR DESCRIPTION
**Pull request by tingtli**
modify upload location to xcat.org for 2.7

### CODE REVIEW

**Comment by tingtli** [comment]
@whowutwut  Hi victor, I just modify 2.7 upload location to xcat.org.
I did the least modification.Please help review it.

**Comment by whowutwut** [comment]
This pull request is in reference to issue #252

File: buildcore.sh [hunk 1]
```diff
@@ -34,7 +34,11 @@
 
 # you can change this if you need to
 UPLOADUSER=litingt
-FRS=/home/frs/project/x/xc/xcat
+USER=xcat
+SERVER=xcat.org
+FILES_PATH="files"
+FRS=/var/www/${SERVER}/${FILES_PATH}
+RELEASE=github.com/xcat2/xcat-core/releases
```
Comment (whowutwut) [inline buildcore.sh]: Are we going to upload 2.7 to github? I think this is not being used but I guess we can leave it in for now to be consistent with the other buildcore.sh 

Or is github release going to point to xcat.org... maybe one less place to maintain if we do it that way.... 
@daniceexi  advice?

**Comment by whowutwut** [comment]
I am ok with the changes

**Comment by whowutwut** [comment]
@tingtli  Do we think we will be creating the 2.7 build?   If so, then I will merge this in...

**Comment by whowutwut** [comment]
Decided in #252 that we will not modify for the 2.7 release, closing for now.

**Pull request closed without merging**